### PR TITLE
Add logo to Readme

### PR DESCRIPTION
**Pull request by kevgo**
@charlierudolph @allewun 

This is the new horizontal logo designed by @redouglas. It looks correctly aligned in the real readme, which you can see [here](https://github.com/Originate/git-town/tree/kg-readme-logo).

### CODE REVIEW

**Comment by charlierudolph** [comment]
I think with the new logo we can get rid of the old title "Git Automation for Agile Development Teams".

**Comment by kevgo** [comment]
Good idea. Removed the slogan, and merged it into the GitHub description of the repo.

**Comment by allewun** [comment]
Looks good, would it be possible to have it in a higher resolution? (for us spoiled retina users)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# _Git Automation for Agile Development Teams_
+<img src="documentation/logo-horizontal.png">
```
Comment (allewun) [inline README.md]: Can we add a title attribute? `<... title="Git Town">`

**Comment by kevgo** [comment]
I like that idea @allewun. @redouglas can you please send me a HiDPI version of the logo, so that we can spoil retina users appropriately?

File: README.md (hunk 1 above)
Comment (kevgo) [inline README.md]: Done

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# _Git Automation for Agile Development Teams_
+<img src="documentation/logo-horizontal.svg" title="Git Town logo">
```
Comment (charlierudolph) [inline README.md]: Git Town

**Comment by kevgo** [comment]
Okay, switched to an SVG version. Github doesn't serve SVG straight from a repo (http://stackoverflow.com/questions/13808020/include-an-svg-hosted-on-github-in-markdown), so I am using the `gh-pages` branch for file hosting. PR for that is coming separately.

@allewun @charlierudolph please add `gh-pages` to your list of non-feature branches.

all comments addressed

**Comment by allewun** [comment]
Looks great! :ship:

**Comment by charlierudolph** [comment]
Awesome :ship:

**Pull request closed without merging**